Doc/c-api/marshal.rst

```diff
@@ -62,16 +57,17 @@ written using these routines?
    for reading.  Only a 16-bit value can be read in using this function,
    regardless of the native size of :c:type:`short`.
 
-   On error, raise an exception and return ``-1``.
+   On error, sets the appropriate exception (:exc:`EOFError`) and returns
+   ``-1``.
 
 
 .. c:function:: PyObject* PyMarshal_ReadObjectFromFile(FILE *file)
 
    Return a Python object from the data stream in a :c:type:`FILE\*` opened for
    reading.
 
-   On error, sets the appropriate exception (:exc:`EOFError` or
-   :exc:`TypeError`) and returns *NULL*.
+   On error, sets the appropriate exception (:exc:`EOFError`, :exc:`ValueError`
```

Comment:
See also two functions below.

Reply:
Done in https://github.com/python/cpython/pull/8457/commits/70bf977328da141267b238602dcf6ee44db8a41a.